client/src/components/Profile/Profile.js

```diff
@@ -0,0 +1,192 @@
+import { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+
+const Profile = () => {
+  const navigate = useNavigate();
```

Comment:
What is this used for? I didn't use this for my navigation stuff but I did set it up a little bit differently.